client/grpc_client.go

```diff
@@ -99,7 +122,7 @@ func (cli *grpcClient) Error() error {
 
 func (cli *grpcClient) EchoAsync(msg string) *ReqRes {
 	req := types.ToRequestEcho(msg)
-	res, err := cli.client.Echo(context.Background(), req.GetEcho())
+	res, err := cli.client.Echo(context.Background(), req.GetEcho(), grpc.FailFast(true))
 	if err != nil {
 		cli.err = err
```

Comment:
I used those functions so itd be less code and easier to update. will fix them according to above comments